content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -5,6 +5,13 @@ reviewers:
 - munnerz
 - enj
 title: Certificates and Certificate Signing Requests
+api_metadata:
+- apiVersion: "certificates.k8s.io/v1"
+  kind: "CertificateSigningRequest"
+  override_link_text: "CSR v1"
```

Comment:
I've included the `override_link_text` feature to customize the default text that appears within the link. This is particularly useful, for instance in the case of HorizontalPodAutoscaler, where we prefer the link to display "v1" and "v2" instead of "HorizontalPodAutoscaler" for both links.